src/actions/datarobot/datarobot.ts

```diff
@@ -0,0 +1,120 @@
+import * as Hub from "../../hub"
+
+import * as httpRequest from "request-promise-native"
+
+const DR_API_URL = "https://app.datarobot.com/api/v2"
+
+export class DataRobotAction extends Hub.Action {
+
```

Comment:
Are these two properties I need to add?

```
supported_download_settings = ["url"]
minimumSupportedLookerVersion = "5.24.0"
```

Reply:
I still need to enhance the `Hub.Action` api to have an enum instead of the `usesStreaming` boolean, so there's not yet an API for specifying you only want URL. (As mentioned in item 1 of https://github.com/looker/actions/pull/187#discussion_r211436837)

Reply:
Got it;)

I've added `minimumSupportedLookerVersion` just like you mentioned and cleanup the code that asserted `downloadUrl`.